admin/src/hooks/useRefreshToken.js

```diff
@@ -0,0 +1,66 @@
+import { setUser } from "@/redux/auth/actions";
+import api from "@/services/api";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { toastr } from "react-redux-toastr";
+import { useHistory } from "react-router-dom";
+
+const PING_INTERVAL = 1000 * 60; // Every minute
```

Comment:
1 min paraît court, en fonction du nombre d'utilisateur connecté cela peux faire de la charge serveur.

Reply:
Complètement d'accord, c'était totalement arbitraire afin de faire mes tests et d'engager la discussion sur les délais que l'on souhaite mettre en place.

Reply:
ça parait bien pour une première itération oui